test/fake-timers-test.js

```diff
@@ -3103,7 +3103,7 @@ describe("FakeTimers", function () {
             assert(typeof date === "string");
         });
 
-        it("creates real Date objects when Date constructor is gone", function () {
+        it.skip("creates real Date objects when Date constructor is gone", function () {
```

Comment:
This test does not seem very useful. I would check the Git history for references.

Reply:
Dug back, found it was just copied over from Sinon ten years ago, so can't be bothered to look deeper:
https://github.com/Oustinger/fake-timers/commit/dbd7d17029f5b258aa14422b74798c844de8bfab#diff-7c9e5e3f3dd46601b73d2d05be43d6d7a33f2f7ccd564494f4eb7b3f4ae915e0R636

The only meaningful bit here about the test is that we check that everything still works as before. So the result of the toString should be the same before and after the removal of the global Date.